pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -602,28 +602,34 @@ self: super: {
   # https://github.com/athanclark/sets/issues/2
   sets = dontCheck super.sets;
 
-  # Install icons, metadata and cli program.
-  # Do not build hgettext as it is broken
-  # https://gitlab.freedesktop.org/bustle/bustle/issues/13
-  bustle = overrideCabal (disableCabalFlag (super.bustle.override { hgettext = null; }) "hgettext") (drv: {
-    buildDepends = [ pkgs.libpcap ];
-    buildTools = with pkgs.buildPackages; [ gettext perl help2man ];
-    patches = [
-      # fix build
-      # https://gitlab.freedesktop.org/bustle/bustle/merge_requests/14
-      (pkgs.fetchpatch {
-        url = "https://gitlab.freedesktop.org/bustle/bustle/commit/ee4b81cbc232d47ba9940f1987777b17452e71ff.patch";
-        sha256 = "0v9cvbmrma5jcqcg1narpm1549h0cg8mr6i00qxmq0x6hs04dnwa";
-      })
-      (pkgs.fetchpatch {
-        url = "https://gitlab.freedesktop.org/bustle/bustle/commit/aae6843f51f54679d440fb3813e61355dc8406b9.patch";
-        sha256 = "1a8hr38hd1gdkqhsy56hyl7njw8ci79iigr81aalkb7hn4ckvh2a";
-      })
-    ];
-    postInstall = ''
-      make install PREFIX=$out
-    '';
-  });
+  bustle = pkgs.lib.pipe super.bustle [
+    # Do not build hgettext as it is broken
+    # https://gitlab.freedesktop.org/bustle/bustle/issues/13
+    (bustle: bustle.override { hgettext = null; })
+    (bustle: disableCabalFlag bustle "hgettext")
+    (bustle: bustle.overrideAttrs (attrs: { meta = builtins.removeAttrs attrs.meta [ "hydraPlatforms" ]; }))
+
+    # Install icons, metadata and cli program.
+    (bustle: overrideCabal bustle (drv: {
+      buildDepends = [ pkgs.libpcap ];
+      buildTools = with pkgs.buildPackages; [ gettext perl help2man ];
+      patches = [
+        # fix build
+        # https://gitlab.freedesktop.org/bustle/bustle/merge_requests/14
+        (pkgs.fetchpatch {
+          url = "https://gitlab.freedesktop.org/bustle/bustle/commit/ee4b81cbc232d47ba9940f1987777b17452e71ff.patch";
+          sha256 = "0v9cvbmrma5jcqcg1narpm1549h0cg8mr6i00qxmq0x6hs04dnwa";
+        })
+        (pkgs.fetchpatch {
+          url = "https://gitlab.freedesktop.org/bustle/bustle/commit/aae6843f51f54679d440fb3813e61355dc8406b9.patch";
+          sha256 = "1a8hr38hd1gdkqhsy56hyl7njw8ci79iigr81aalkb7hn4ckvh2a";
+        })
+      ];
+      postInstall = ''
```

Comment:
does `installPhase = null;` work to use the regular make stuff in `setup.sh`?

Reply:
We still want to use Haskell install phase since the makefile only covers non-Haskell parts.